modules/h2/src/main/java/org/h2/expression/aggregate/AggregateDataCollecting.java

```diff
@@ -153,17 +155,16 @@ public Value getSharedArgument() {
 
     /** {@inheritDoc} */
     @Override public void cleanup(Session ses) {
-        H2MemoryTracker memTracker;
-        if (values != null && (memTracker = ses.memoryTracker()) != null) {
+        if (values != null && tracker != null) {
```

Comment:
Let's help GC in anyway

if (values != null)
   values = null;
if (tracker != null)
   tracker.release ....